integ-test/src/test/java/org/opensearch/sql/legacy/PluginIT.java

```diff
@@ -70,8 +70,12 @@ public void sqlDeleteSettingsTest() throws IOException {
             "{\n"
                 + "  \"error\": {\n"
                 + "    \"reason\": \"Invalid SQL query\",\n"
-                + "    \"details\": \"DELETE clause is disabled by default and will be deprecated. Using "
-                + "the plugins.sql.delete.enabled setting to enable it\",\n"
+                + "    \"details\": \""
+                    + "Query request is not supported. Either unsupported fields are present, the "
+                    + "request is not a cursor request, or the response format is not supported.\\n"
```

Comment:
Why `\\n` rather than `\n`?

Reply:
It creates `\n` in the inner string. There is a string in the string, which contains multiline message.